app/Transformers/UserTransformer.php

```diff
@@ -50,6 +50,7 @@ class UserTransformer extends Fractal\TransformerAbstract
         'unranked_beatmapset_count',
         'unread_pm_count',
         'user_achievements',
+        'ranking_expanded',
```

Comment:
something generic like 'user_preferences' since it should be able to have multiple settings under the key